tests/test_custom_scheme_registry.py

```diff
@@ -15,6 +15,7 @@
 
 import pytest
 
+from idutils.normalizers import to_url
```

Comment:
weird that the gh action warns when this is good by all account :open_mouth:

Reply:
Yes, the depreciation warning shows up for any import even if it's correct.  It's very annoying but I don't know of a way to fix it. https://github.com/inveniosoftware/idutils/blob/fe319c4361b9c9ce6d171c471627ce598552ac4e/idutils/__init__.py#L23